ci/provisioning/post_provision_config_nodes_EL_7.sh

```diff
@@ -46,32 +46,32 @@ EOF
 
     # Mellanox OFED hack
     if ls -d /usr/mpi/gcc/openmpi-*; then
+        version=$(rpm -q --qf %{version} openmpi)
```

Comment:
(lint) This { is literal. Check expression (missing ;/\n?) or quote it. [SC1083]
(lint) This } is literal. Check expression (missing ;/\n?) or quote it. [SC1083]